[linux] Add Linux Kernel

Evernow opened the pull request:
Sources used for this can be found here: https://www.kernel.org/category/releases.html

Evernow committed:
Create Linux Kernel

Sources used for this can be found here: https://www.kernel.org/category/releases.html

hebbet commented on products/linuxkernel.md:
```diff
@@ -0,0 +1,59 @@
+    release: 2016-12-11
+    eol: 2023-01-01
+    
+  - releaseCycle: "4.14 KTS"
```
There is a small typo

Evernow replied:
Sorry about that, fixed. Also fixed issues with the description of releases, looks good now.

Evernow committed:
Formatting fix

Evernow committed:
KTS -> LTS

Evernow committed:
Changed formatting of description.

Evernow committed:
Command to check kernel version

captn3m0 committed:
Mark LTS releases as lts: true

- mark LTS releases using lts: true
- Shorter description
- Remove maintainer names (not really relevant to those outside of the kernel dev community).

Evernow committed:
Merge branch 'endoflife-date:master' into master

Evernow committed:
Clarified the default is two years for LTS releases
